### PR TITLE
feat(/submission): added my own submission

### PR DESCRIPTION
**Pull request by engrshege**
This is my submission to the ZTM Job Board.

Thanks.
Best Regards,
Oluwasegun.

### CODE REVIEW

**Comment by engrshege** [comment]
Hello ZTM!
Still awaiting approval for the merge request please.

**Comment by engrshege** [comment]
Hello ZTM!
Still awaiting approval for the merge request please.

**Comment by engrshege** [comment]
Hello ZTM!
Still awaiting approval for the merge request please.

File: Submissions/engrshege.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "[Oluwasegun Idowu]",
+  "img": "[https://avatars.githubusercontent.com/u/52855925?v=4]",
+  "email": "[engrshege@gmail.com]",
+  "links": {
+    "website": "[https://engrshege.github.io/myportfolio/]",
```
Comment (ProsperousHeart) [inline Submissions/engrshege.json:6]: none of these should have brackets around the URLs

File: Submissions/engrshege.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "linkedin": "https://www.linkedin.com/in/engrshege",
+    "github": "https://github.com/engrshege"
+  },
+  "jobTitle": "[Frontend Engineer]",
+  "location": {
+    "city": "[Ikotun]",
+    "state": "[Lagos]",
+    "country": "[Nigeria]"
```
Comment (ProsperousHeart) [inline Submissions/engrshege.json:14]: Please also remove the brackets from here

**Review by ProsperousHeart — CHANGES_REQUESTED** [review]
Please remove brackets from requested locations

File: Submissions/engrshege.json (hunk 2 above)
Comment (engrshege) [reply Submissions/engrshege.json]: I have now made corrections by removing the brackets.

Thank you so much for your review ma'am.

File: Submissions/engrshege.json (hunk 1 above)
Comment (engrshege) [reply Submissions/engrshege.json]: I have removed the brackets from here too ma'am.

Thank you ma

**Pull request merged**